app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -112,6 +119,9 @@ define([
   NewsletterSticky.prototype._handleScroll = function() {
     if (this._hasScrolledOverPercentage(this.config.appearsAfterPercentage)) {
       this.$el.addClass(this.config.visibleClassName);
+
+      this.previousFocusElement = $(':focus');
```

Comment:
This effectively does this ```$('*:focus')``` which will scan the DOM, I wonder if using ```document.activeElement``` is as widely accepted?